src/PrometheusPushGateway/PushGateway.php

```diff
@@ -79,9 +79,9 @@ public function delete(string $job, array $groupingKey = []): void
      * @param string $method
      * @throws GuzzleException
      */
-    private function doRequest(CollectorRegistry $collectorRegistry, string $job, array $groupingKey, $method): void
+    private function doRequest(?CollectorRegistry $collectorRegistry, string $job, array $groupingKey, $method): void
```

Comment:
This shouldn't be optional.

Reply:
The `delete` function calls doRequest with a `null` value and because we are enforcing strict types using `declare(strict_types=1);` it breaks if we don't make it optional.

Reply:
Good catch, could you please relfect this also in the docstring?

Reply:
Happy to.